docs(anoncomms): adding oracle track and furps (#392)
logos-co/roadmap: content/anoncomms/furps/oracle.md

# Oracle FURPS

## Functionality

1. Oracle nodes fetch price data from predefined sources
2. Each oracle node signs its observation and participates in a simple off-chain consensus with a threshold
3. The agreed value is written to the LEZ contract
4. The LEZ contract exposes basic functions to update and read the latest price

## Usability

> 1. The system design is documented in an RFC

Reviewer: ```suggestion
1. The system design is documented in a LIP
```
Trying to get used to new terminology :)

> 1. The system design is documented in an RFC

Reviewer: For the sake of consistent terminology, could we call "system design" here something something _"protocol"_? E.g."the system protocol is documented in a LIP specification"

> 2. The oracle nodes is implemented in Rust

Reviewer: ```suggestion
2. The oracle nodes are implemented in Rust
```
